docs/development/developer_onboarding.md

```diff
@@ -8,7 +8,7 @@ However, before running `php artisan rogue:setup`, you are going to need a few p
 
 ### Northstar
 
-For the most part, you can use the QA Northstar URL for your development environment: `https://northstar-qa.dosomething.org`
+For the most part, you can use the QA Northstar URL for your development environment: `https://identity-dev.dosomething.org`
```

Comment:
Let's also say "you can use the Development Northstar URL" here, instead of "QA".

Reply:
done! good suggestion!